docs/source/setup_docker.rst

```diff
@@ -59,6 +59,85 @@ Building and Configuring Containers
     Don't forget that you need to reset your default username and password if you are going
     to use these Docker images in production mode!
 
+Using Docker for Development
+----------------------------
+
+The development environment is configured for live reloading (ie restart webserver when files change)
+and debugging. It builds off the base docker-compose.yml, so it's necessary
+to specify the files being used in docker-compose commands as seen below.
+
+Build
+^^^^^
+
+.. code-block:: bash
+
+    # create volumes for the database and media directory
+    docker volume create --name=seed_pgdata
```

Comment:
based on the docker file shouldn't this be in the form of {name}-seed_pgdata

Reply:
Not necessarily, as mentioned above we're templating in the volume name now, with a default value of `seed_pgdata`